corehq/motech/repeaters/expression/repeater_generators.py

```diff
@@ -49,11 +49,9 @@ def get_url(self, repeat_record, url_template, payload_doc):
 class ArcGISFormExpressionPayloadGenerator(ExpressionPayloadGenerator):
 
     def get_url(self, repeat_record, url_template, payload_doc):
-        if not (
-            toggles.UCR_EXPRESSION_REGISTRY.enabled(repeat_record.domain)
-            and toggles.ARCGIS_INTEGRATION.enabled(repeat_record.domain)
-        ):
+        if not toggles.ARCGIS_INTEGRATION.enabled(repeat_record.domain):
             return ""
+        return super().get_url(repeat_record, url_template, payload_doc)
```

Comment:
:flushed: :thinking:  So this was only working (not raising a `TypeError`) because none of the ArcGIS interations defined a URL template.

Reply:
I guess so